backend/src/main/game/cards/test_cards.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.main.game.cards.card import Card
+from backend.src.main.game.cards.cards import Cards
+
+
+def test_cards_starts_empty():
+    assert Cards().cards.__len__() == 0
```

Comment:
Should not be calling __len__
either use `assert not Cards().cards` or `assert len(Cards().cards) == 0`